app/Http/Controllers/ProcessController.php

```diff
@@ -23,7 +24,10 @@ public function process(Request $request)
     	$message = $request->input('message');
     	//$recipient = $request->input('recipient');
         //split the recipient email address by comma delimiter
+        //allows for multiple recipients
         $recipients = explode(",", $request->input('recipient'));
+        $emailCheck = $request->input('emailCheck');
+        $smsCheck = $request->input('emailSms');
```

Comment:
We may want to hold off on committing these until we're using them -- I think `$emailCheck` is attempting to get a value using the `id` while `$smsCheck` is using the `name`.